src/path.util.ts

```diff
@@ -21,9 +21,30 @@ export const getValuePropertyPair = (instance: any, path: string):
         decodeURIComponent(validPathSegments[validPathSegments.length - 1]) : undefined
   };
 };
-
-export const toDataPath = (path: string): string =>
-  path.split('/').filter(subPath => PATH_SEGMENTS_TO_IGNORE.indexOf(subPath) === -1).join('/');
+const toDataPathSegments = (path: string): Array<string> => {
+  const segments = path.split('/');
+  const startFromRoot = segments[0] === '#' || segments[0] === '';
+  return segments.filter((segment, index) => {
```

Comment:
Rather first check for `startFromRoot` and formulate simplified filter expressions.